package.json

```diff
@@ -21,7 +21,7 @@
     "create-package": "yarn nx g @fluentui/workspace-plugin:react-library",
     "format": "node scripts/executors/src/format.js",
     "generate-version-files": "node -r ./scripts/ts-node/src/register ./scripts/generators/src/generate-version-files",
-    "postinstall": "node ./scripts/package-manager/src/postinstall.js",
+    "postinstall": "node ./scripts/package-manager/src/postinstall.js && yarn patch-package",
```

Comment:
can we switch those ? our custom postinstall should be ran after node_modules are set